arrows/ocv/resection_camera.cxx

```diff
@@ -205,15 +202,42 @@ ::resection(
 
   auto const reproj_error = d_->reproj_accuracy;
 
-  auto const err =
-    cv::calibrateCamera(
+  auto err = std::numeric_limits<double>::infinity();
+  vital::matrix_3x3d K = cal->as_matrix();
+  cv::TermCriteria term_criteria{
+    cv::TermCriteria::COUNT + cv::TermCriteria::EPS,
+    d_->max_iterations, DBL_EPSILON };
+  using MatD = cv::Mat_<double>;
+  MatD cv_K;
+  eigen2cv( K, cv_K );
+  auto const dc0 = dist_coeffs;
+  auto const focal_scales = {0.5, 1.0, 2.0};
+  auto focal_scale = 0.0; // preferred focal scale after optimization
```

Comment:
I have a preference for comments on the line before a statement rather than at the end of a line after the statement, but it seems our style guide doesn't comment on this, so I guess it's okay.  The style guide does say to start the comment with a capital letter.

Reply:
done